src/commands/base/CommandUsage.ts

```diff
@@ -0,0 +1,3 @@
+export default interface CommandUsage {
```

Comment:
Consider making this part of the base command

Reply:
Only some commands have a usage, e.g. IgnoreCommand since additional input is required. Way to go = Make optional in BaseCommand and only set in those that have it?